app/services/sockethub-xmpp.js

```diff
@@ -94,8 +94,8 @@ export default Ember.Service.extend({
   handleJoinCompleted(space, message) {
     const channelId = message.target['@id'].split('/')[0];
     const channel = space.get('channels').findBy('sockethubChannelId', channelId);
-    if (!isEmpty(channel)) {
-      channel.set('connected', true);
+    if (channel) {
+      this.observeChannel(space, channel);
```

Comment:
I don't see where the channel is set to connected in this PR. It looks like the observa

Reply:
The channel gets set to connected when receiving the channel's user list here: https://github.com/67P/hyperchannel/blob/fix/127-xmpp_room_user_list/app/services/coms.js#L175

Reply:
Hmm, I think it's a bit unexpected that setting the user list sets the channel to connected as well.

Reply:
Aight.